src/App.js

```diff
@@ -1,37 +1,17 @@
 import React from 'react';
-const PROGRAM_TITLE = "Todo List";
-const TODO_LIST = [
-  {
-    id: 1,
-    title: "Begin To-Do app",
-  },
-  {
-    id: 2,
-    title: "Checkout new github branch"
-  },
-  {
-    id: 3,
-    title: "Create list"
-  },
-  {
-    id: 4,
-    title: "Display list on page"
-  },
-  {
-    id: 5,
-    title: "Push changes to github"
-  }
-];
+import TodoList from "./TodoList";
+import AddTodoForm from "./AddTodoForm";
 
-function App() {
-  return (
+const PROGRAM_TITLE = 'Todo List';
```

Comment:
Nitpick (doesn't require changes):
Since this variable is only used once within the `h1` tag, I'd recommend putting it inline:
```jsx
<h1>Todo Title</h1>
```
If this title needs to be used multiple times within this file, then it _may_ make more sense to turn it into a reusable constant. That said, things like a todo list, though it may only be used once, may be easier to read when they're not inline. 

There aren't any rules for in-lining data types versus extracting them into a reusable constant variable outside a component; it mostly boils down to personal preference and legibility.